src/components/album/Album.js

```diff
@@ -75,4 +76,11 @@ function Album({ album, wishListId }) {
   );
 }
 
+Album.propTypes = {
+  album: PropTypes.shape({
+    _id: PropTypes.string.isRequired,
+  }).isRequired,
+  wishListId: PropTypes.string,
+};
```

Comment:
This is good; While it's not necessarily needed to put in _all_ possible fields of the album object;  I would recommend at least putting in the types of all the fields that are used in this file.

Reply:
We can further validate using custom [PropType validation,](https://designcode.io/react-hooks-handbook-custom-proptypes) that for example `album.releaseDate` is a string in the expected date format, or that `averageRating` is a number between 1 and 5 as we would expect